integration/test/basic/basic_test.go

```diff
@@ -63,25 +59,57 @@ func TestChartConfigChartsInstalled(t *testing.T) {
 		return
 	}
 
-	guestNamespace := "giantswarm"
 	guestG8sClient := g.G8sClient()
 
 	// Wait for chart configs as they may not have been created yet.
 	err := waitForChartConfigs(guestG8sClient)
 	if err != nil {
 		t.Fatalf("could not get chartconfigs %v", err)
 	}
+}
+
+func TestChartConfigPatchDeploy(t *testing.T) {
+	// These versions have no chartconfigs so we return early.
+	clusterOperatorVersion := os.Getenv("CLOP_VERSION_BUNDLE_VERSION")
+	if clusterOperatorVersion == "0.1.0" || clusterOperatorVersion == "0.2.0" {
```

Comment:
Why are we doing this here? I think it would be better to stop doing these things and rather not execute tests in the first place in case we have the option to not do that. Isn't it also that these versions are not tested anymore anyways because there are way newer versions?

Reply:
Just took what was already there for the other tests. Not really sure about all the release versions and what is out there and not. Do you have an idea @rossf7?

Reply:
I think we now get the cluster operator version from the release index. So lets remove these. But doing it in a follow up PR is fine with me.